examples/rbd/storageclass.yaml

```diff
@@ -33,7 +33,8 @@ parameters:
    # creation (thin provisioning is the default).
    thickProvision: "false"
    # (required) RBD image features, CSI creates image with image-format 2
-   # CSI RBD currently supports only `layering` feature.
+   # CSI RBD currently supports `layering`, `journaling`, `exclusive-lock`
+   # features. If `journaling` is enabled, must enable `exclusive-lock` too.
```

Comment:
This should be a comma separated list, without spaces, I think? Might be good to have an example, or mention it explicitly.

Reply:
Yes, it should be comma separate without space.